internal/aghtls/aghtls.go

```diff
@@ -28,3 +33,14 @@ func SaferCipherSuites() (safe []uint16) {
 
 	return safe
 }
+
+func UserPreferredCipherSuites(ciphers []string) (userCiphers []uint16) {
```

Comment:
1.  The name of this function isn't really informative.  I feel like a better name would be something like `ParseCipherIDs`.

2.  This function requires documentation.

Reply:
changes as per the recommendation